letsdrill.rb

```diff
@@ -1,12 +1,12 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  grade_levels = [ [ 90, "A" ], [80, "B"], [70, "C"], [60, "D"] ]
```

Comment:
I thought about that, but the algorithm only works if I'm working through the grades in decreasing order. I didn't think I could assume that while walking the hash.